cmake/FindMbedTLS.cmake

```diff
@@ -0,0 +1,13 @@
+find_path(MBEDTLS_INCLUDE_DIRS mbedtls/ssl.h)
```

Comment:
Copyright?

Reply:
done & tweaked the notice script accordingly